logs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Does ignoring /logs/* from the main file not accomplish just this?

Reply:
git wont create an empty dir

Reply:
We should probably do this in whatever is specifying the log output... but okay.